beamLogo.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
```

Comment:
**question:** Did we need to import React? I sometimes see this is a warning, but never an error. Typically it can be ignored.

Reply:
no I don't think so!